### PR TITLE
Rework Liberal assassination section

### PR DESCRIPTION
**Pull request by 7thsanctum**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -140,18 +140,17 @@ Chain investigations in 9/10 player games are almost always a bad idea. They lim
...
 
-It is difficult to identify Hitler over the course of the game. If you have suspicion of someone for being Hitler you should be sure before assassinating as Hitler could be disguised as the most Liberal player instead.
+There are only two ways to know for sure if someone is fascist:
+* Investigation
+* Be in direct dispute over a policy
 
-For identifying Hitler for assassination, semi-valid reasons for assassinations can include but are not limited to the following:
+It is often safest to shoot whoever you are in a dispute with as a result of investigations of policy disputes. They may be less likely to be Hitler and this may not grant you much trust with others but the Liberals majority will increase. The alternative is to aim for someone else with the aim of shooting Hitler. The hard part is identifying them. The following reasons are possible reasons you may try to shoot Hitler:
```
Comment (TartanLlama) [inline index.md]: whomever

Comment (TartanLlama) [inline index.md]: as a result of investigations or policy disputes

Comment (TartanLlama) [inline index.md]: They are less likely to be Hitler, so other Liberals may question your choice, but increasing the Liberal majority is worthwhile.

Comment (TartanLlama) [inline index.md]: The alternative is to go gunning for Hitler instead; the hard part is identifying them.

Comment (TartanLlama) [inline index.md]: Some reasons to suspect someone of being Hitler are:

Comment (NeuralSandwich) [inline index.md]: > whomever

While technically correct, I have read many cases that it doesn't matter. This is more prescriptive and descriptive grammar these days.

However, I don't agree with this tactic. Although I have done this a bunch of times, I don't think shooting someone you are in a dispute with makes sense.

Comment (TartanLlama) [inline index.md]: Why wouldn't it make sense? You would be absolutely certain of killing a Fascist, which is about the best you can do without much other information.

Comment (NeuralSandwich) [inline index.md]: True, in a bunch of situations where I have been in that position I became untrusted.

If you have gotten into a conflict it is *likely* not Hitler. So based on that

| Players | Other Fascist + Hitler | Chance of killing Hitler | Chance of killing another Fascist (who is not Hitler) |
| --- | --- | --- | --- |
| 5 | 1 | 33.33% | 33.33% |
| 6 | 1 | 25 % | 0-25% |
| 7 | 2 | 20% | 20-40% | 
| 8 | 2 | 16% | 16.67-33.33% |
| 9 | 3 | 14% | 28.57-42.86% |
| 10 | 3 | 11 % | 25-37.5% |

For the 5 player game, Hitler might be more outlandish

Comment (TartanLlama) [inline index.md]: I'd argue that in most situations I'd rather get the 100% Fascist kill, especially in an odd player game. Maybe we should mention taking into account the number of players when making your decision.

Comment (NeuralSandwich) [inline index.md]: I think this doesn't really make sense in 5-6/ player game.

**Comment by TartanLlama** [comment]
@NeuralSandwich what other safe options would you have? An option is only ever safe if you *know* someone is Fascist, which you only know if you're in a dispute. Perhaps we should differentiate between a dispute from a policy play (less likely to be Hitler) and a dispute from an investigation (reasonable chance of being Hitler).

File: index.md (hunk 1 above)
Comment (TartanLlama) [inline index.md]: If you guess wrong as a Liberal in a 5 player game then you pretty much insta-lose.

Comment (NeuralSandwich) [inline index.md]: There should be a caveat for known not Hitler

Comment (TartanLlama) [inline index.md]: How about we rephrase this section as a series of considerations for whether you should shoot a known fascist or hunt for Hitler?

Comment (NeuralSandwich) [inline index.md]: Yeah, I think this needs to be reworked a little... Although I obviously feel like we might make it a bit too complicated (maybe this needs more testing?)

I was basing this on the idea that you gave your Chancellor a choice and they played Fascist. If you were just given the power (say you drew 3 Fascist) and you only knew a Fascist from a dispute or investigation then *yes, absolutely kill that person*. If the dispute comes from the Fascist just played and they gave you the assassination power... This is where the table comes I I believe.

I feel like a state machine might be needed here.

Comment (7thsanctum) [inline index.md]: There is a lot here to take in without much consensus. I've fixed the other comments but will need to work on this part a bit more.

Comment (TartanLlama) [inline index.md]: Cool, I'll merge this in for now and make an issue to track other changes.

**Comment by TartanLlama** [comment]
@NeuralSandwich you cool to merge these changes for now and have a new issue to track rewording?

**Pull request merged**